eighteen/src/main/kotlin/com/lynbrookrobotics/kapuchin/subsystems/drivetrain/DrivetrainComponent.kt

```diff
@@ -46,4 +48,8 @@ class DrivetrainComponent(hardware: DrivetrainHardware) : Component<DrivetrainCo
         leftLazyOutput(value.left)
         rightLazyOutput(value.right)
     }
+
+    init {
+        if (clock is Ticker) clock.realtimeChecker(hardware.jitterPulsePin::set, { hardware.jitterReadPin.period.Second })
```

Comment:
`jitterReadPin` is a WPI [`Counter`](http://first.wpi.edu/FRC/roborio/release/docs/java/edu/wpi/first/wpilibj/Counter.html).
`Counter` has as `getPeriod()` method which returns a `double` in seconds.
In Kotlin, we call this method with `jitterReadPin.period`, so we have to convert the result to a `Quan` ourselves.

Reply:
Yep, realized this after I got the IDE to work properly.